server/api-manager.js

```diff
@@ -939,7 +943,7 @@ export default MindbodyAccess;
 var client = new mindbodyQueries();
 client.getAuth()
   .then((value) => {
-    client.AUTH_TOKEN = value.AccessToken;
+    client.authToken = value.AccessToken;
```

Comment:
This is in a section of comments describing how to use the class. The comments were out of date with the new variable name.